### PR TITLE
Added how to configure and use conditions on Workflows in Umbraco Forms

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

Added how to configure and use conditions on Workflows in Umbraco Forms

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Forms v10,12,13,14

## Deadline (if relevant)

Anytime

### CODE REVIEW

File: 10/umbraco-forms/editor/attaching-workflows/README.md [hunk 1]
```diff
@@ -48,7 +48,19 @@ Once the Workflow Type has been selected, you will need to configure the workflo
 
 To use data from the submitted Form in your workflow, head over to the [Magic Strings](../../developer/magic-strings.md) article and learn more about how that's done.
 
-Fill in the settings and hit **Submit**. The workflow is added to your Form and it will be shown at the bottom of the page.
+### Configuring Condition on a Workflow
+
+You can apply conditions to a workflow that trigger it only under specific circumstances. After adding the desired workflow type (for example, sending an email), you can add a condition to the workflow.
```
Comment (sofietoft) [inline 10/umbraco-forms/editor/attaching-workflows/README.md:53]:
```suggestion
You can apply conditions to a workflow to ensure it only triggers under specific circumstances. You could as an example enable a condition that enables sending an email only if an email is added.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
This is written as a guide.
Shouldn't it be added to a list instead? 🤔

**Comment by eshanrnh** [comment]
@sofietoft, I wrote it as a guide to match the rest of the article. Changing this part to a step-by-step or list format means we'll have to change the entire article to keep it consistent 🙈 The other articles in the **Editor** section are also written as guides, so if we decide to change this article to a step-by-step or list format, we might have to consider those as well.

**Comment by sofietoft** [comment]
> @sofietoft, I wrote it as a guide to match the rest of the article. Changing this part to a step-by-step or list format means we'll have to change the entire article to keep it consistent 🙈 The other articles in the **Editor** section are also written as guides, so if we decide to change this article to a step-by-step or list format, we might have to consider those as well.

Alright, that makes sense 😁

I do think thought that we at some point should ensure that whenever we have guides with steps, that the steps should be put into ordered lists. It's much easier to follow 💪🏻

**Review by sofietoft — APPROVED** [review]

**Comment by eshanrnh** [comment]
Yup, I totally agree 💪 Merging this PR.

**Pull request merged**